org.d3cod3.Mosaic.yml

```diff
@@ -253,11 +253,19 @@ modules:
       - cd apps/d3cod3/Mosaic && make -j$FLATPAK_BUILDER_N_JOBS Release
       - mkdir /app/mosaic
       - cp -ar * /app/mosaic
+      - install -Dm644 org.d3cod3.Mosaic.appdata.xml /app/share/appdata/org.d3cod3.Mosaic.appdata.xml
+      - install -Dm644 org.d3cod3.Mosaic.desktop /app/share/applications/org.d3cod3.Mosaic.desktop
+      - install -Dm644 apps/d3cod3/Mosaic/bin/data/images/logo_128.png /app/share/icons/hicolor/48x48/apps/org.d3cod3.Mosaic.png
```

Comment:
Not sure why you would install a 128px icon as 48? we do require a 128px anyway.

Reply:
Because I mistyped it 😅 

I'll fix it tomorrow. Let's hope this build ends well